docs/release/trg-2/trg-2-5.md

```diff
@@ -2,23 +2,27 @@
 title: TRG 2.05 - Repository metafile
 ---
 
-| Status     | Created     | Post-History       |
-|------------|-------------|--------------------|
+| Status     | Created     | Post-History           |
+|------------|-------------|------------------------|
+| Update     | 10-Jan-2024 | reword the why section |
 | Active     | 24-Apr-2023 |
-| Prerelease | 7-Mar-2023  | Moved out of draft |
-| Draft      | 24-Feb-2023 | Draft release      |
+| Prerelease | 7-Mar-2023  | Moved out of draft     |
+| Draft      | 24-Feb-2023 | Draft release          |
 
 ## Why
 
-Due to many products having more than one repository, we need a way of identifying dependent repositories automatically. This allows us to analyze repositories for quality checks and for generating documentation across repositories.
+Many products are build from more than one repository.
+We aim to have automatic quality checks and therefore need a metadata file to identify the repositories belonging to a product.
 
 ## Description
 
-To enable automatic analysis of repositories, all repositories **MUST** have a file named `.tractusx` present on root-level.
+To enable automatic analysis of repositories, all repositories **MUST** have a file named `.tractusx` present on root-level with the dedicated `repoCategory` which points to the displayed section for the repository.
```

Comment:
Hi @FaGru3n,
actually 'repoCategory' isn't mandatory, it defaults to 'product' but can be set to 'special' or 'support' when required. 

Also [this line](https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/blob/0a562e02995908e769ac06cf60aa0673006374e5/docs/release/trg-2/trg-2-5.md?plain=1#L34C1-L34C122) looks weird, I think it should be:

Optional field. By default it will be treated as regular product repo. Available options: "special", "support".

Reply:
[1831649](https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/pull/581/commits/18316494d3afbaf6d8c6d92d36bcac785ceba4d9)